src/test/scala/edu/berkeley/ce/rockslicing/SeedJointSelectorSpec.scala

```diff
@@ -0,0 +1,212 @@
+package edu.berkeley.ce.rockslicing
+
+import org.scalatest.FunSuite
+
+class SeedJointSelectorSpec extends FunSuite {
+
+  private def compareJoints(joint1: Joint, joint2: Joint): Boolean = {
```

Comment:
I added an `approximateEquals` method to the `Joint` as used that in the unit tests instead.